enterprise/server/raft/replica/replica.go

```diff
@@ -763,18 +739,68 @@ func (sm *Replica) split(wb *pebble.Batch, req *rfpb.SplitRequest) (*rfpb.SplitR
 		return nil, err
 	}
 
-	// update the left local range (when this batch is committed).
-	if err := sm.rangeCheckedSet(wb, constants.LocalRangeKey, leftRDBuf); err != nil {
+	splitRequiresExternalChanges := sm.splitAppliesToThisReplica(req)
+	if splitRequiresExternalChanges {
+		sm.log.Debugf("Split %+v applies to this replica c%dn%d: making external changes!", req, sm.clusterID, sm.nodeID)
+		// If this split already happened and we're just replaying the
```

Comment:
This comment block seems misplaced since here we're handling the actual split here. Maybe move it either above the IF or in the else?